diesel/src/expression/expression_methods/global_expression_methods.rs

```diff
@@ -295,11 +295,10 @@ pub trait ExpressionMethods: Expression + Sized {
     ///     use self::posts::dsl::{posts, author_name};
     ///     let connection = establish_connection();
     ///
-    ///     let data = users.inner_join(posts)
+    ///     let data: QueryResult<Vec<String>> = users.inner_join(posts)
     ///         .filter(name.nullable().eq(author_name))
```

Comment:
We should probably decide on style guidelines for when chaining functions together across multiple rows. One function per row? As many as will fit within the soft 80 char limit?